frontend/compositions/header/header.module.css

```diff
@@ -0,0 +1,85 @@
+.wrapper {
+  width: 100%;
+  height: 124px;
```

Comment:
Can we use `rem` over `px`? Important for zoom responsiveness for low vision users

If it merges ahead of you, feature58 will introduce CSS size variables that will manage that conversion. 
But if not, or if you'd like to just do it now, basically divide your `px` number by 16 & that'll be your `rem` number!

Reply:
@srsexton94 yes I will convert from px to rem! thank you for your review + feedback.